[DOCS-13939] Update Synthetics Test Details Page screenshots and add AI failure summaries (#36272)
DataDog/documentation: content/en/synthetics/browser_tests/test_results.md

## Overview

The test details page opens after a Synthetic browser test executes and is organized into four tabs: [Activity](#test-activity), [Test Runs](#test-runs), [Performance](#test-performance), and [Properties](#test-properties). Use these tabs to monitor uptime, inspect individual runs, review aggregate performance metrics, and manage test configuration. When a run fails, see [Failed results](#failed-results) for troubleshooting tools such as AI failure summaries and screenshot comparison.

## Test activity

On the **Activity** tab, you can see:

- The **Global Uptime** graph, which displays the total uptime of all test locations in a given time interval. The global uptime visualization displays red only if the [alert conditions][20] configured for a test are triggered in the given time interval. Since location uptime is computed based on the final test result after retries complete, [fast retry][24] intervals directly impact what appears in your total uptime graph. For more information about uptime monitoring, see the [Website Uptime Monitoring with SLOs][14] guide.

> - A **Timeline** of alert triggers, recoveries, and summary periods.

Reviewer: ```suggestion
- A **Timeline** of alert triggers, recoveries, and test modifications.
```

There's a pinned summary tab at the top but I'm not sure how to describe it.

## Test runs

On the **Test Runs** tab, you can see all individual runs of your test. Filter by status (passed or failed), run type, location, or device, and click any row to inspect that run in detail.

{{< img src="synthetics/browser_tests/synthetics_test_runs.png" alt="The Test Runs tab on a browser Test Details page showing a filterable table of test runs with status, date, run type, steps, duration, location, device, browser, and test version columns" style="width:100%" >}}

Browser test runs include components such as [screenshots](#screenshots-and-actions), [page performance data](#test-performance), [errors](#errors-and-warnings), [resources](#resources), and [backend traces](#backend-traces) to help troubleshoot your [test failure](#failed-results).

{{% collapse-content title="Test run columns" level="h4" %}}

The following describes each column in the **Test Runs** table:

Status
: The status of the test run (`PASSED` or `FAILED`).

Date
: The relative time and timestamp when the run executed.

Run Type

> : The type of test run (scheduled, CI, fast retry, or manually triggered).

Reviewer: ```suggestion
: The type of test run (scheduled, CI, or manually triggered).
```

Technically you cannot filter on fast retry or not for the test runs. You can just see if a scheduled run is a "fast retry" or not

<img width="367" height="199" alt="Image" src="https://github.com/user-attachments/assets/18873a95-1216-4f11-a37f-679430b6357d" />

**Downtimes**
: Lists [scheduled downtimes][27] that pause execution of this test, for example during planned maintenance windows.

**Configuration as Code**
: Exports the test configuration in formats such as Terraform for managing tests as code.

{{% /collapse-content %}}

## Failed results

A test result is considered `FAILED` if it does not satisfy its assertions or if a step failed for another reason. You can troubleshoot failed runs by looking at their screenshots, checking for potential [errors](#errors-and-warnings) at the step level, and looking into [resources][17] and [backend traces](#backend-traces) generated by their steps.

### AI failure summaries

When a browser test run fails, Datadog generates an AI failure summary to help you identify the cause and next steps for investigation. Each summary includes:

- A short explanation of what failed, grounded in run data such as network errors, assertions, and screenshots.
- A classification of the failure as either a **true failure** (a real problem with your application) or a **test misconfiguration** (an issue with the test setup).
- Suggested next steps for troubleshooting.

> AI failure summaries appear on the test run details page for any failing browser test run. Treat them as a starting point for investigation, not as authoritative root cause analysis, because LLM-generated content can contain inaccuracies.

Reviewer: Do we want to include a sentence asking user to give us feedback with the 👍 / 👎  button? (and tell us more)

Author: I had that in there initially then removed it :) I think it's a good idea!

## Test events

> Alerts from your Synthetic test monitors appear on the timeline in the [**Activity** tab](#test-activity), where you can review alert triggers, recoveries, and summary periods alongside the global uptime graph. To search for alerts from Synthetic tests in the Events Explorer, navigate to [**Events** > **Explorer**][18] and enter `Event Type:synthetics_alert` in the search query. For more information, see [Using Synthetic Test Monitors][13].

Reviewer: I'm not sure to understand what "summary periods" mean

Reviewer: ```suggestion
Alerts from your Synthetic test monitors appear on the timeline in the [**Activity** tab](#test-activity), where you can review alert triggers, recoveries, and summary periods alongside the global uptime graph. To search for alerts from Synthetic tests in the Events Explorer, navigate to [**Events** > **Explorer**][18] and enter `@evt.type:synthetics_alert` in the search query. For more information, see [Using Synthetic Test Monitors][13].
```
The one used here did not work when I copied and pasted it into the explorer
